docs/adv-rumors.rst

```diff
@@ -5,15 +5,14 @@ adv-rumors
     :summary: Improves the conversation menus in Adventure mode.
     :tags: adventure interface
 
-In adventure mode, start a conversation with someone and then run this tool
-to improve the "Bring up specific incident or rumor" menu and all other similar menus. This tool will:
+In adventure mode, start a conversation with someone. This tool will:
 
-- Adds all words to keywords for easier filtering/searching everywhere
-- slay/slain/slew keywords for all relevant instances
+- Add all words to keywords for easier filtering/searching everywhere
+- Additional 'slay' keyword for all relevant instances
```

Comment:
what are the relevant instances? the previous code appeared to detect `slew` specifically in the text as an indication that the adventurer was the slayer. Is that still the case?